packages/components/alert-dialog/src/AlertDialogAction.tsx

```diff
@@ -0,0 +1,13 @@
+import { Dialog, DialogCloseProps } from '@spark-ui/dialog'
+import { ElementRef, forwardRef } from 'react'
+
+export type AlertDialogActionElement = ElementRef<typeof Dialog.Close>
+export type AlertDialogActionProps = DialogCloseProps
+
+export const AlertDialogAction = forwardRef<AlertDialogActionElement, AlertDialogActionProps>(
+  (props, ref) => {
+    return <Dialog.Close ref={ref} data-spark-component="alert-dialog-action" {...props} />
```

Comment:
I think this is a mistake to use `Dialog.Close` for an `Action` subcomponent ? 

Aside from that I think we don't need to expose `AlertDialog.Action` because it does not add anything to the button. I checked on Radix, it doesn't add any html attribute or behaviour.